tests/functional/e/.#emacs_file_lock_redefined_conf.py

```diff
@@ -1,3 +1,3 @@
-# [invalid-name]
+# [invalid-name, non-ascii-file-name]
```

Comment:
What is non-ascii about this file name? The `#`?

Reply:
Yep the # is not valid for the regex.

But this might be duplicated to the invalid-name checker.
Maybe we can remove the regex all togehter and only use the Python internal .ascii checker if that's what's wanted. I am not sure.
But looking at it, it makes kind of sense...

Reply:
@DanielNoord 
The regex used is `self._non_ascii_rgx_compiled = re.compile("[^A-Za-z0-9_]")`

Maybe we can only use: `Py37Str(check_string).isascii()` -> that is actually what we want to check. Everything else should be catched by `invalid name` probably.
But to be honest looking at the source code for `invalid_name` I don't get it.
You have more background knowledge @DanielNoord, so let me know if we drop the regex for the `non-ascii-*` checker.

Reply:
I would be fine with using `isascii()` and dropping the `regex`. Seems to be less cumbersome.

@Pierre-Sassoulas What's your opinion?

Reply:
I just removed it. Too much complexity that is not needed.